crates/solver/src/solver/score_computation.rs

```diff
@@ -51,7 +51,16 @@ impl ScoreCalculator {
             - self.gas_price_factor * gas_price / 10_000_000_000.
             - self.nmb_orders_factor * nmb_orders as f64;
         let success_probability = 1. / (1. + exponent.exp());
-        let score = success_probability * (surplus + fees - gas_amount * gas_price);
+        let obj = surplus + fees - gas_amount * gas_price;
+        let cap = 10000000000000000.0;
```

Comment:
Do you see `cap` changing it's value any time soon? If so, it might be worth reading it from configuration.

Reply:
It's not clear, as we are currently considering different options. Tbh, and given that this fix was long overdue (as Gnosis solvers underbidding has affected certain auctions), I would keep it as is for now.